dnsapi/dns_me.sh

```diff
@@ -114,7 +114,7 @@ _get_root() {
     fi
 
     if _contains "$response" "\"name\":\"$h\""; then
-      _domain_id=$(printf "%s\n" "$response" | cut -c 2- | head -c -2 | sed 's/{.*}//' | sed -r 's/^.*"id":([0-9]+).*$/\1/')
+      _domain_id=$(printf "%s\n" "$response" | sed 's/^{//; s/}$//; s/{.*}//' | sed -E 's/^.*"id":([0-9]+).*$/\1/')
       if [ "$_domain_id" ]; then
```

Comment:
could you please not use `-E` option with `sed` ?
I'm afraid it's not supported on some platforms.

Thanks.

Reply:
I've committed the requested change, reverting to `-r` instead of `-E`
I had thought `-E` was the more compatible option of the two